frontend/src/HOC/AuthGuard.js

```diff
@@ -49,11 +50,15 @@ export const AuthGuard = (WrappedComponent) => {
 
     render() {
       if (this.props.keycloak) {
-        if (this.props.isAuthenticated) {
+        if (this.props.isAuthenticated && this.props.userAccessData.includes("mds-mine-view")) {
```

Comment:
instead of "mds-mine-view" 

use: 
```
import { USER_ROLES } from '@/constants/environment';
this.props.userAccessData.includes(USER_ROLES.role_view)
```